msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -476,6 +736,11 @@ public PublicClientApplicationConfiguration getConfiguration() {
         return mPublicClientConfiguration;
     }
 
+    @Override
+    public Boolean getIsSharedDevice() {
```

Comment:
How about renaming to : `isSharedDevice` ?

Reply:
Hmm... curious when we're allowed to deviate from standardard getter/setter... is there a guide we are following?

Reply:
Made the change anyway....